### PR TITLE
Adding development FAQ section to DEVELOPMENT.md

### PR DESCRIPTION
**Pull request by beckerhe**
This PR adds some frequently asked questions and their answers to the DEVELOPMENT.md file.

These discussed questions mainly relate to the conan introduction and the changes in
workflow which this introduction resulted in.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -67,6 +67,138 @@ A `.clang-format` file which defines our specific code style lives in the
 top level directory of the repository. The style is identical to the Google
 style.
 
+## FAQ
+
+### What's the difference between `bootstrap-orbit.{sh,bat}` and `build.{sh,bat}`?
+
+`bootstrap-orbit.{sh,bat}` performs all the task which have to be done once per developer machine.
```
Comment (ronaldfw) [inline DEVELOPMENT.md]:
```suggestion
`bootstrap-orbit.{sh,bat}` performs all the tasks which have to be done once per developer machine.
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -67,6 +67,138 @@ A `.clang-format` file which defines our specific code style lives in the
...
+This includes:
+* Installing system dependencies
+* Installing conan if necessary.
+
+Afterwards `bootstrap-orbit.{sh,bat}` calls `build.{sh,bat}`.
+
+`build.{sh,bat}` on the other hand performs all the tasks which have to be done
+once per build configuration. It creates a build directory, named after the
+given conan profile, installs the conan-managed dependencies into this build folder,
+calls cmake for build configuration and starts the build.
```
Comment (ronaldfw) [inline DEVELOPMENT.md]: This implies that when one changes the git branch, one may have to rerun build.sh, right? Maybe add a sentence about that?

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -67,6 +67,138 @@ A `.clang-format` file which defines our specific code style lives in the
...
+```bash
+cd <build_folder>/
+cmake --build . # On Linux
+cmake --build . --config {Release,RelWithDebInfo,Debug} # On Windows
+```
+
+Alternatively, you can also just call `make` on Linux. Check out the next section on how to
+enable building with `ninja`.
+
+### How do I enable Ninja for my build?
```
Comment (ronaldfw) [inline DEVELOPMENT.md]: Nit: "ninja" for consistency with the rest?

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -67,6 +67,138 @@ A `.clang-format` file which defines our specific code style lives in the
...
+
+> **Note:** Linux only for now! On Windows you have to use MSBuild.
+
+If you want to use `ninja`, you cannot rely on the `build.sh` script, which automatically
+initializes your build with `make` and that cannot be changed later on easily.
```
Comment (ronaldfw) [inline DEVELOPMENT.md]:
```suggestion
initializes your build with `make` and that cannot be changed easily later on.
```

File: DEVELOPMENT.md (hunk 2 above)
Comment (beckerhe) [reply DEVELOPMENT.md]: Yes. I added a sentence explaining that!

**Comment by beckerhe** [comment]
Thanks Ronald for reviewing this. I really appreciate it!

**Pull request merged**